engine/runtime-with-polyglot/src/test/scala/org/enso/interpreter/test/instrument/RuntimeVisualizationsTest.scala

```diff
@@ -1,10 +1,8 @@
 package org.enso.interpreter.test.instrument
 
-import org.enso.distribution.FileSystem
+import org.apache.commons.io.FileUtils
 import org.enso.distribution.locking.ThreadSafeFileLockManager
```

Comment:
This test shall verify JavaScript is enabled somehow.

Reply:
I enabled the test `run internal IDE visualisation preprocessor catching error` that runs JS